kratos/python_scripts/time_based_ascii_file_writer_utility.py

```diff
@@ -132,35 +134,40 @@ def __ValidateAndAssignOutputFolderPath(self):
+        if self.path_type == "relative":
+            if self.folder_name != "":
+                if raw_path != "":
+                    # assign the default value
+                    self.folder_name = "TimeBasedAsciiResults"
 
-                warn_msg  = 'Relative path "'+ raw_path +'" contained wrongly in "file_name": "'+ self.file_name +'"\n'
-                warn_msg += 'Use parameter "folder_name" to specify correctly\n'
-                warn_msg += 'Using the default relative path "' + self.folder_name + '" instead'
-                KratosMultiphysics.Logger.PrintWarning("TimeBasedAsciiFileWriteUtility", warn_msg)
+                    warn_msg  = 'Relative path "'+ raw_path +'" contained wrongly in "file_name": "'+ self.file_name +'"\n'
+                    warn_msg += 'Use parameter "folder_name" to specify correctly\n'
+                    warn_msg += 'Using the default relative path "' + self.folder_name + '" instead'
+                    KratosMultiphysics.Logger.PrintWarning("TimeBasedAsciiFileWriteUtility", warn_msg)
 
-            subfolders = os.path.normpath(self.folder_name).split(os.sep)
+                subfolders = os.path.normpath(self.folder_name).split(os.sep)
 
-            absolute_folder_path = os.getcwd()
-            relative_folder_path = ""
-            for folder in subfolders:
-                absolute_folder_path = os.path.join(absolute_folder_path, folder)
-                relative_folder_path = os.path.join(relative_folder_path, folder)
-
-            self.file_name = os.path.join(relative_folder_path, raw_file_name)
+                absolute_folder_path = os.getcwd()
+                relative_folder_path = ""
+                for folder in subfolders:
+                    absolute_folder_path = os.path.join(absolute_folder_path, folder)
+                    relative_folder_path = os.path.join(relative_folder_path, folder)
 
-        else:
-            if raw_path != "":
-                warn_msg  = 'Relative path "'+ raw_path +'" contained wrongly in "file_name": "'+ self.file_name +'"\n'
-                warn_msg += 'Use the parameter "folder_name" to specify correctly\n'
-                warn_msg += 'Using the current directory instead'
-                KratosMultiphysics.Logger.PrintWarning("TimeBasedAsciiFileWriteUtility", warn_msg)
+                self.file_name = os.path.join(relative_folder_path, raw_file_name)
 
+            else:
+                if raw_path != "":
+                    warn_msg  = 'Relative path "'+ raw_path +'" contained wrongly in "file_name": "'+ self.file_name +'"\n'
+                    warn_msg += 'Use the parameter "folder_name" to specify correctly\n'
+                    warn_msg += 'Using the current directory instead'
+                    KratosMultiphysics.Logger.PrintWarning("TimeBasedAsciiFileWriteUtility", warn_msg)
+                absolute_folder_path = os.getcwd()
+                self.file_name = raw_file_name
+
+        elif self.path_type == "PyCOMPSs":
```

Comment:
Why not absolute?
```suggestion
        elif self.path_type == "absolute":
```

Reply:
I don't think absolute is the correct name, since the user specifies a relative name and the absolute path is added by PyCOMPSs.

Reply:
However, yes, the user can set an absolute name and that will be an absolute path, so I have nothing against the change.